### ISSUE TITLE
BioJS DNA logo is left-handed and wrong

### ISSUE THREAD

**Issue by daviddao** [biojs/biojs#158]
After showing BioJS to scientist from the Broad Institute, they realized that the DNA inside the logo is left-handed while DNA in nature is right-handed. 

![f04-05](https://cloud.githubusercontent.com/assets/1241240/14005339/aab6c086-f138-11e5-8b6d-4a8cf4aa8d94.JPG)

![](https://avatars1.githubusercontent.com/u/5148092?v=3&s=200)

We should consider removing the DNA from the logo to stay consistent with our corporate design on the website: https://biojsnet.herokuapp.com/

**Comment by daviddao** [comment]
@greenify any comments on that?

**Comment by greenify** [comment]
Quite astonished to realize that :D

How about mirroring the DNA in the SVG and we should be good to go? @daviddao do you have time to do that?

> We should consider removing the DNA from the logo to stay consistent with our corporate design on the website

I think you know that I personally don't like the plain green circle - especially because I know how it [was created](https://drive.google.com/drive/u/0/folders/0BwdiZp7qSaBhRktiNFE1Y0NNdVk)
-> I would prefer if we could stick with the DNA

**Comment by AgrawalAmey** [comment]
I guess [this](https://drive.google.com/a/bits-pilani.ac.in/file/d/0BwyWFeou7mr2cTdlcF9zV253NGs/view?usp=sharing) this should sort the problem. Also the logo now weighs 5kbs instead of 44kbs.

**Comment by daviddao** [comment]
@AgrawalAmey we don't have permission for that

**Comment by daviddao** [comment]
@jessica-jordan @manuelcorpas what are your thoughts about the logo?

**Comment by AgrawalAmey** [comment]
@daviddao It was just implementation of suggestion by @greenify .

**Comment by greenify** [comment]
daviddao means that we can't access your file due to your file permissions.

**Comment by AgrawalAmey** [comment]
Sorry for inconvenience,I have fixed the link.

**Comment by rajido** [comment]
I think it is bit bias to genetics/genomics to use the DNA. Why not a protein, an antibody, a cell or a plant? I like more the plain logo but I might be bias as well.

**Comment by jessica-jordan** [comment]
@daviddao thanks for the great comment. I personally like the dna in the logo, as it makes the graphic harder to confuse with the recycling brand logo. 

Still, I see the point that it wouldn't have to be DNA in particular, but what could we use instead?

**Comment by daviddao** [comment]
@jessica-jordan I agree with @rajido! A cell would be very nice! :) It is beautiful under microscope, can be seen as an abstract module of a bigger organism and therefore fits BioJS a lot.

**Comment by jessica-jordan** [comment]
@Davidson @rajido I like the cell idea a lot. Is someone of you proficient with Photoshop? If not, I'll also try my luck with that

**Comment by daviddao** [comment]
:+1: I am looking forward to your design :)

**Comment by rajido** [comment]
@jessica-jordan  ... I think it is a good sign the first thing that comes to your mind when you look at the current logo is the idea of recycling/reusing. It might be worth a try the idea of the cell. However I would try to keep it simple and ideally similar since for better or worse people already identifies the logo with BioJS. 

Instead of DNA, protein, eukaryotic cell, virus or anything specific of one particular domain of biology I would try with something more common used in bioinformatics, something like annotations. In bioinformatics we have annotations for everything ... genes, proteins, phenotypes, cells ... Something that could play well with the current logo would be annotations similar to those represented by the circos visualisation ... http://circos.ca/images/samples/
